src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -491,33 +491,46 @@ public static void TCP_ServiceCertExpired_Throw_MessageSecurityException()
 
     [Fact]
     [OuterLoop]
+    // Confirm that the Validate method of the custom X509CertificateValidator is called and that an exception thrown there is handled correctly.
     public static void TCP_ServiceCertFailedCustomValidate_Throw_Exception()
     {
         string testString = "Hello";
+        NetTcpBinding binding = null;
+        EndpointAddress endpointAddress = null;
+        ChannelFactory<IWcfService> factory = null;
+        IWcfService serviceProxy = null;
```

Comment:
NIT: why the assignment to null, then reassignment afterwards? Just directly use 

```
var binding = new NetTcpBinding(); 
endpointAddress = new EndpointAddress(new Uri(Endpoints.Tcp_VerifyDNS_Address), new DnsEndpointIdentity(Endpoints.Tcp_VerifyDNS_HostName));  
```

etc. 
Not blocking for merge, just looks a little strange to me.